pkg/meta/redis.go

```diff
@@ -4240,3 +4247,49 @@ func (m *redisMeta) doAttachDirNode(ctx Context, parent Ino, dstIno Ino, name st
 		return err
 	}, m.inodeKey(parent), m.entryKey(parent)))
 }
+
+// caller makes sure inode is not special inode.
+func (m *redisMeta) touchAtime(ctx Context, ino Ino, cur *Attr) (rerr syscall.Errno) {
+	if (m.conf.AtimeMode != StrictAtime && m.conf.AtimeMode != RelAtime) || m.conf.ReadOnly {
+		return 0
+	}
+
+	var attr *Attr
+	newAttr := &Attr{}
+	if cur != nil {
+		attr = cur
+	} else if m.of.Check(ino, newAttr) {
+		attr = newAttr
+	}
+
```

Comment:
check `atimeNeedsUpdate` here first, and return directly if the update is not needed.

Reply:
Fixed